src/main/java/com/rocketden/main/dto/room/UpdateSettingsRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.rocketden.main.dto.room;
+
+import com.rocketden.main.dto.user.UserDto;
+import com.rocketden.main.model.Difficulty;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UpdateSettingsRequest {
+    private UserDto initiator;
+    private Difficulty difficulty;
```

Comment:
In the future when we have more settings for the host to toggle, would there only be one request that contains values for every type of setting, or a different request for each one?

Reply:
That's a good question - my thought would be that it'd all be in one request that way we wouldn't have a bunch of very similar endpoints and associated tests, but I'm okay either way

Reply:
Yeah I think that's a good idea, preferably it'd be just one endpoint, I think that'd be a simpler setup.